### DISCUSSION TITLE
SF has disappeared from Safari Extensions

### DISCUSSION THREAD

**Discussion by XJZY7MI** [gildas-lormeau/SingleFile#1303] [Q&A]
**Describe the bug**
SF has disappeared from Safari Extensions

**To Reproduce**
1. Go to Safari 
2. Click on Preferences
3. Click on Extensions  
4. SF not present under Extensions

**Expected behavior**
See SF listed as an option under Extensions

**Screenshots**
 

**Environment**
 - OS: Big Sur Version 11.7.10 (20G1427)
 - Browser: Safari
 - Version: 16.6.1 (16615.3.12.11.5, 16615)

**Additional context**
It was working fine until last week. I received a few updates for SF since I installed it and never had a problem. Is it possible  Apple updated something in the OS that I'm not aware of?

**Comment by gildas-lormeau** [comment]
That's strange. It looks like SingleFile has been somehow uninstalled on your machine. Did you try to install it again?

**Comment by XJZY7MI** [comment]
The app is in my Application folder. When I click on it I get the following  
![SF1](https://github.com/gildas-lormeau/SingleFile/assets/75222304/bbed4a10-e525-4448-8663-97cbf75e74ef)

But when I check Preferences/Extensions it is not listed ....

**Comment by XJZY7MI** [comment]
Dumb question: if its an Extension, why is it in the Application folder?

**Comment by gildas-lormeau** [comment]
I cannot reproduce your issue on my machine but it's running on the latest macOS/Safari versions. However, SingleFile should work fine in Safari 16.x. For the record, here is a screenshot of the "Extensions" in the Safari settings. I installed the extension from the store.

<img width="836" alt="image" src="https://github.com/gildas-lormeau/SingleFile/assets/396787/9fc68145-3a8d-406d-8560-599cbfaab830">


The extension is packaged as an Application because that is the only way to distribute extensions on Safari.

**Comment by XJZY7MI** [comment]
Thanks for the follow-up! I really like this tool and find it very useful. Oddly enough, I found the following while searching Github - https://discussions.apple.com/thread/253409221   

 I may give it a shot. I will let you know what happens.
